src/WhereConditions/WhereHasConditionsDirective.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace Nuwave\Lighthouse\WhereConditions;
+
+use GraphQL\Error\Error;
+use GraphQL\Language\AST\FieldDefinitionNode;
+use GraphQL\Language\AST\InputValueDefinitionNode;
+use GraphQL\Language\AST\ObjectTypeDefinitionNode;
+use Illuminate\Support\Str;
+use Nuwave\Lighthouse\Schema\AST\Codegen;
+use Nuwave\Lighthouse\Schema\AST\DocumentAST;
+use Nuwave\Lighthouse\Schema\AST\PartialParser;
+use Nuwave\Lighthouse\Schema\Directives\BaseDirective;
+use Nuwave\Lighthouse\Support\Contracts\ArgBuilderDirective;
+use Nuwave\Lighthouse\Support\Contracts\ArgManipulator;
+use Nuwave\Lighthouse\Support\Contracts\DefinedDirective;
+
+class WhereHasConditionsDirective extends BaseDirective implements ArgBuilderDirective, ArgManipulator, DefinedDirective
+{
+    /**
+     * @var \Nuwave\Lighthouse\WhereConditions\Operator
+     */
+    protected $operator;
+
+    /**
+     * WhereHasConditionssDirective constructor.
+     *
+     * @param  \Nuwave\Lighthouse\WhereConditions\Operator  $operator
+     * @return void
+     */
+    public function __construct(Operator $operator)
+    {
+        $this->operator = $operator;
+    }
+
+    public static function definition(): string
+    {
+        return /* @lang GraphQL */ <<<'SDL'
+"""
+Add a dynamically client-controlled relationship WHERE condition to a fields query.
+The argument it is defined on may have any name but **must** be
```

Comment:
That is actually no longer true, can you remove that from `@whereConditions` too?

Reply:
Thanks, removed it.